renovate.json

```diff
@@ -0,0 +1,20 @@
+{
+    "extends": [
+        "config:base"
+    ],
+    "schedule": [
+        "before 8am"
+    ],
+    "reviewers": [
+        "agelsomini",
+        "caloisio",
+        "lmaxwell24"
+    ],
+    "labels": ["dependencies"],
+    "packageRules": [
+        {
+            "matchPackagePatterns": ["^robotpy|^wpilib"],
```

Comment:
Pyntcore? Pyfrc?

Reply:
Good catch, would be nice if the naming was more consistent...